src/librustc_borrowck/borrowck/mod.rs

```diff
@@ -1299,9 +1299,10 @@ impl<'a, 'tcx> BorrowckCtxt<'a, 'tcx> {
                         snippet
                     );
                 } else {
-                    db.span_label(
+                    db.span_suggestion(
```

Comment:
Use `span_suggestion_with_applicability` and `Applicability::MachineApplicable`, so that `rustfix` and VScode can apply this automagically.